pkg/controllers/disruption/controller.go

```diff
@@ -294,6 +293,6 @@ func (c *Controller) logInvalidBudgets(ctx context.Context) {
 		}
 	}
 	if buf.Len() > 0 {
-		logging.FromContext(ctx).Errorf("detected disruption budget errors: %s", buf.String())
+		log.FromContext(ctx).Error(nil, "detected disruption budget errors: %s", buf.String())
```

Comment:
I wonder if it's possible to convert that `buf.String()` into an error so that we aren't passing a null error to the error function